mantle/kola/harness.go

```diff
@@ -106,6 +106,9 @@ var (
 	extTestNum  = 1 // Assigns a unique number to each non-exclusive external test
 	testResults protectedTestResults
 
+	nonexclusivePrefixMatch = regexp.MustCompile(`^non-exclusive-test-bucket-[0-9]/`)
+	nonexclusiveMatch       = regexp.MustCompile(`^non-exclusive-test-bucket-[0-9]$`)
```

Comment:
```suggestion
	nonexclusiveWrapperMatch = regexp.MustCompile(`^non-exclusive-test-bucket-[0-9]$`)
```

Does this one just match the nonexclusive wrapper?